docs/source/usage/modes/search.rst

```diff
@@ -94,10 +94,7 @@ The following commands are available in search mode
 
 .. describe:: toggletags
 
-    flip presence of tags on this thread.
-    A tag is considered present if at least one message contained in this
-    thread is tagged with it. In that case this command will remove the tag
-    from every message in the thread.
+    flip presence of tags on this thread. A tag is considered present if at least one message contained in this thread is tagged with it. In that case this command will remove the tag from every message in the thread.
```

Comment:
I just saw that this file is auto generated. Forget my comment.

(damn I hate all these auto generated files that we have to keep in git)